### ISSUE TITLE
Module '"spacetime"' has no exported member 'Spacetime'. Did you mean to use 'import Spacetime from "spacetime"' instead?

### ISSUE THREAD

**Issue by antonstefer** [spencermountain/spacetime#407]
### Setup

package.json:
```
{
    "type": "module"
    "dependencies" : {
        "spacetime": "7.5.0",
        "typescript": "5.0.4"
    }
}
```

tsconfig.json:
```
{
    "extends": "@tsconfig/node20/tsconfig.json",
    "compilerOptions": {
        "lib": ["es2023"],
        "module": "node16",
        "moduleResolution": "node16"
        "target": "es2022",

        "strict": true,
        "esModuleInterop": true,
        "skipLibCheck": true,
        "forceConsistentCasingInFileNames": true,
        "useUnknownInCatchVariables": false,
        "noUnusedLocals": true,
        "resolveJsonModule": true,

        "outDir": "dist/",
        "baseUrl": ".",
        "verbatimModuleSyntax": true
    },
    "exclude": ["node_modules"],
}
```

### Problem
Importing Spacetime like this:
```
import type { Spacetime } from 'spacetime'
import spacetime from 'spacetime'
```
produces the following Typescript error:
```
TS2614: Module '"spacetime"' has no exported member 'Spacetime'. Did you mean to use 'import Spacetime from "spacetime"' instead?
```
Directly importing from `spacetime/types/types` like mentioned [here](https://github.com/spencermountain/spacetime/wiki/Typescript) does not work as well:
```
TS2307: Cannot find module 'spacetime/types/types' or its corresponding type declarations.
```
Is there another way to import the types or is something broken?
This syntax has been working when my project was still using CommonJS.

Thank you!

**Comment by jecraig** [comment]
Can you create a small repo that recreates this issue.

**Comment by antonstefer** [comment]
Here you go:
https://github.com/antonstefer/spacetime-import-issue

**Comment by spencermountain** [comment]
Thank you Anton! Will keep this open until it's released, in the next few days.
cheers

**Comment by spencermountain** [comment]
released as `7.6.0`
cheers!
